contrib/api-stress-test/roles/openpai/install/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+docker_registry_namespace: openpai
+docker_registry_domain: docker.io
+docker_registry_tag: stresstest
+
+#docker_registry_username: example
+#docker_registry_password: example
+
+kube_config_path: "{{local_home_path}}/.kube/config"
+
+openpai_branch_name: "stress-test"
```

Comment:
Why not use `master` branch. Do we plan to maintain the branch?

Reply:
No, but this branch has been modified for vk scheduling. And can be used directly.

Reply:
Can we doc about this? If we want to use latest code to do stress test, want steps need to follow.
I think we may need to:
1. checkout master branch
2. apply code change. (Maybe make it as a patch is more user-friendly, but manual modify is OK)
3. push the change to 'stress-test' branch or other branch
4. Change `openpai_branch_name`, run script

Reply:
Master branch always changes. I think apply patch to master branch is not good.

Reply:
Stress-test can only be a case or example for user who wanna try this stress test environment. And if user wanna use it to test his own code. I still suggest to change the code based on his branch. And I think the commit example is enough for the developer.

Reply:
If the user just want to try this environment. Driectly running ansible-playbooks is ok.

Reply:
Got it, thanks